### PR TITLE
Add padding to the feedback widget #748  -->  Fixed

### PR DESCRIPTION
**Pull request by LynxSumit**
🚀 PR #748 Update: Added padding to the feedback widget 🚀

🔧 Changes Made:
Hi team! I'm excited to share my latest contribution to our project. In this PR, I've addressed issue #748 by adding padding to the feedback widget. This improvement not only enhances the visual aesthetics of the widget but also ensures a better user experience.

🛠️ What I've Done:
- Added appropriate padding to the feedback widget.
- Conducted thorough testing to confirm that the padding works as expected.
- Updated relevant documentation to reflect these changes.

🧐 Why This Matters:
This PR improves the overall user interface by providing a more polished and user-friendly appearance to the feedback widget. This small but meaningful change aligns with our commitment to delivering a top-notch user experience.

🌟 Let's Collaborate:
I'm looking forward to your feedback and any suggestions for further improvement. Please feel free to review and provide your thoughts. Together, we can ensure that our project continues to shine!

🙏 Thank you for your time and consideration!

### CODE REVIEW

File: site/src/util/code-snippets-map.json
Comment (angiejones) [inline site/src/util/code-snippets-map.json:1]: this file should not be updated

File: site/src/contributors.json [hunk 1]
```diff
@@ -58,5 +58,11 @@
     "login": "alec-brooks",
     "avatar_url": "https://avatars.githubusercontent.com/u/1539341?v=4",
     "contributions": ["doc"]
+  },
+  {
```
Comment (angiejones) [inline site/src/contributors.json:62]: @EbonyLouis are you ok with this?

File: site/src/components/Card.jsx [hunk 2]
```diff
@@ -4,8 +4,9 @@ import React from 'react';
 function Card({ children, className }) {
   return (
     <div
-      className={`flex flex-col bg-transparent border-[#282828] border-2 shadow overflow-hidden sm:rounded-lg lg:p-10 sm:p-5 ${className}`}
+      className={`flex flex-col bg-transparent border-[#282828] border-2 shadow overflow-hidden sm:rounded-lg p-8 lg:p-10  sm:p-5 ${className}`}
```
Comment (angiejones) [inline site/src/components/Card.jsx:7]:
```suggestion
      className={`flex flex-col bg-transparent border-[#282828] border-2 shadow overflow-hidden sm:rounded-lg p-8 lg:p-10 sm:p-5 ${className}`}
```
removed extra space

**Review by angiejones — CHANGES_REQUESTED** [review]

File: site/src/components/Card.jsx (hunk 2 above)
Comment (angiejones) [reply site/src/components/Card.jsx:7]: and revert code-snippets-map.json, thats the most important change to make

**Comment by angiejones** [comment]
@LynxSumit thanks so much! would you like us to wait until Monday to merge this so that you get credit for Hacktoberfest?

File: site/src/contributors.json (hunk 1 above)
Comment (EbonyLouis) [reply site/src/contributors.json:62]: yes that's fine.

**Review by angiejones — CHANGES_REQUESTED** [review]
@LynxSumit there's merge conflicts with code-snippets-map.json. Please completely remove any changes to this file from your PR

File: site/src/util/code-snippets-map.json
Comment (angiejones) [inline site/src/util/code-snippets-map.json:1]: are you able to just remove this file from your PR? i'm not sure why it's being automatically updated

**Review by angiejones — APPROVED** [review]

**Comment by angiejones** [comment]
merging this one @EbonyLouis. can you add to Leaderboard?

**Pull request merged**